src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -0,0 +1,21 @@
+import {addMonths, format, isBefore} from 'date-fns';
```

Comment:
@amyevans Does it makes sense to add tests for this utils?

Reply:
Yes I think that'd be good!

Reply:
tests added